wolfcrypt/src/port/st/stm32.c

```diff
@@ -361,5 +361,372 @@ int wc_Stm32_Aes_Init(Aes* aes, CRYP_InitTypeDef* cryptInit,
 #endif /* WOLFSSL_STM32_CUBEMX */
 #endif /* WOLFSSL_AES_DIRECT || HAVE_AESGCM || HAVE_AESCCM */
 #endif /* !NO_AES */
-
 #endif /* STM32_CRYPTO */
+
+#ifdef WOLFSSL_STM32_PKA
+#include <stm32wbxx_hal_conf.h>
+#include <stm32wbxx_hal_pka.h>
```

Comment:
If you are going to use uint8_t and uint32_t consider adding stdint.h include (although I'm sure stm32wb includes are doing it). Or use our wolf types byte and word32.